impedance/plotting.py

```diff
@@ -16,8 +18,7 @@ def _set_orderOfMagnitude(self, range):
 
 
 def plot_nyquist(ax, Z, scale=1, units='Ohms', fmt='.-', **kwargs):
```

Comment:
What is the expected behavior of "scale" it doesn't seem to change anything?

Reply:
In the past it set the formatting of the x and y axes (to show 10^3 for example and fixing #12) in these lines     https://github.com/ECSHackWeek/impedance.py/blob/8a29e65099279fc489f1f54892327dee67328e50/impedance/plotting.py#L67-L73

Not sure why it's not working any more...

Reply:
The work around to get this to work originally is here: https://stackoverflow.com/a/3679918

It's not working for me anymore either however...

Reply:
Looks like it was a recent change in matplotlib that [added a new way of doing this](https://matplotlib.org/users/prev_whats_new/whats_new_3.0.html#ability-to-scale-axis-by-a-fixed-order-of-magnitude). Think it should work now (although I'm still not sure this is the best way of handling widely varying impedances) :smile: